app/controllers/Editor.scala

```diff
@@ -45,7 +56,7 @@ final class Editor(env: Env) extends LilaController(env) {
   def data =
     Open { implicit ctx =>
       fuccess {
-        val situation = readFen(get("fen"))
+        val situation = readFen(get("fen"), Variant.libStandard(GameLogic.Chess()))
```

Comment:
should this always be chess, or take in get("variant") and also amend the corresponding route and json data for the editor.